### DISCUSSION TITLE
嵌套

### DISCUSSION THREAD

**Discussion by Zwahxf** [alibaba/formily#2962] [Q&A 2.x]
SchemaField 中的FormTab ，FormTab.TabPane中有个选择框的联动，官网给的例子是在SchemaField中配schema参数，但是外层的tab是SchemaField，SchemaField又不能嵌套，如何解决呢

**Comment by giligiliduang** [comment]
我现在遇到自定义Field内部嵌套新的 SchemaForm第一次初始化值会变成undefined

**Comment by blakeyi** [comment]
嵌套太多层了,好卡
